eclass/elisp-common.eclass

```diff
@@ -228,7 +231,7 @@ elisp-emacs-version() {
 		# Redirecting stdin and unsetting TERM and DISPLAY will cause
 		# most of them to exit with an error.
 		unset TERM DISPLAY
-		${EMACS} ${EMACSFLAGS} -l "${T}"/emacs-version.el </dev/null
+		${tmout} ${EMACS} ${EMACSFLAGS} -l "${T}"/emacs-version.el </dev/null
 	)
 	ret=$?
 	rm -f "${T}"/emacs-version.el
```

Comment:
maybe `|| die`? I see there's handling for the error case, but purely for consistency

Reply:
Error handling is just below in the `if` block (checking `$?`). I'd rather not move any nontrivial code into `$( )`.

Also, `$( )` spawns a subshell, and `die` won't work there in EAPI 6.